superx/routes/signup.py

```diff
@@ -85,6 +98,7 @@ def login():
         user = User.query.filter_by(email=form.email.data).first()
         if check_password_hash(user.password, form.password.data):
             login_user(user)
+            session['cart'] = current_user.city
```

Comment:
should this be session['city'] instead of session['cart']?

Reply:
yah

Reply:
yah. my bad